src/ngrest/base/ActiveButton.php

```diff
@@ -154,7 +155,29 @@ public function getCondition()
     {
         return empty($this->_condition) ? '' : $this->_condition;
     }
-
+    
+    private $_permissionLevel = null;
+    
+    /**

```

Comment:
When its a method from the interface you can use write `{@inheritdoc}` then IDE's will use the inherited doc and also the luya.io api renderer.

Reply:
```
/**
 * {@inheritDoc}
 */
```

Is enough then.

Reply:
OK